### PR TITLE
Fixes #247: Added CompareTwitterProfiles App

### LINKED ISSUE #247: Add Twitter Profiles Comparison App
- Issue type: Feature request 

### Short description
The app will show comparison between two twitter profiles for instance it shows Number of followers, statuses etc.

### Would you like to work on the issue?
Yes

### PR DESCRIPTION
**Pull request by kavithaenair**
### Short description
Related to issue #247 
Demo link: https://kavithaenair.github.io/compareTwitterProfiles/

### Screenshots
image

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by kavithaenair** [comment]
@djmgit @vibhcool @singhpratyush @Achint08 @SKrPl Please review.

File: compareTwitterProfiles/css/style.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#searchBar
+{
```
Comment (djmgit) [inline compareTwitterProfiles/css/style.css]: put this opening brace on the previous line.

File: compareTwitterProfiles/index.html [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+                       placeholder="Enter User 2" class="form-control"
+                       ng-init="query1 = ''"
+                       ng-keyup="$event.keyCode == 13 && query1 != '' && query != '' ? Search() : null">
+                     </td>
+                   </tr>
+                 </table>
+                 <span class="input-group-btn">
+                   <button href='#' class="btn btn-default"
+                     type="button" id="button"
+                     ng-click="query != '' ? Search() : null">Compare
```
Comment (djmgit) [inline compareTwitterProfiles/index.html]: Did you intentionally leave out query1?

File: compareTwitterProfiles/css/style.css [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+.post-wrapper .post-inner p {
+  font-weight: 300;
+  padding: 10px 0;
+}
+
+.post-wrapper .post-inner p.lead {
+  font-size: 1.2em;
+}
+
+.Emoji.Emoji--forText {
```
Comment (djmgit) [inline compareTwitterProfiles/css/style.css]: Where this style has been used?

File: compareTwitterProfiles/css/style.css [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  width: 70%;
+}
+
+th, td {
+  padding: 8px;
+  text-align: center;
+  border-bottom: 1px solid #ddd;
+}
+
+.post-wrapper:first-child .post-inner {
```
Comment (djmgit) [inline compareTwitterProfiles/css/style.css]: Where is the post-wrapper class used?

File: compareTwitterProfiles/index.html (hunk 2 above)
Comment (kavithaenair) [inline compareTwitterProfiles/index.html]: I didn't see that. Will update the code.

File: compareTwitterProfiles/index.html [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </center>
+    <script>
+        var app = angular.module('TweetSearch', []);
```
Comment (Achint08) [inline compareTwitterProfiles/index.html]: A separate file for Controller would look good. Any reason for adding a controller in the HTML itself?

**Comment by kavithaenair** [comment]
@djmgit @Achint08 Please re-review.

File: compareTwitterProfiles/script.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
+var app = angular.module('TweetSearch', []);
+app.controller('Controller', ['$scope', '$http', '$sce', function ($scope, $http, $sce) {
+
+    $scope.Search = function () {
+
+        var userQueryCommand = 'http://api.loklak.org/api/user.json?' +
```
Comment (djmgit) [inline compareTwitterProfiles/script.js]: empty line

File: compareTwitterProfiles/index.html [hunk 7]
```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html ng-app='TweetSearch'>
+
```
Comment (djmgit) [inline compareTwitterProfiles/index.html]: empty line

File: compareTwitterProfiles/index.html [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+<head>
+    <link rel="icon" type="image/png" href="../../artwork/favicon.png">
+    <title>Comapre User Profile Page</title>
+    <link href="/css/loklak.css" rel="stylesheet">
+    <script src="/js/jquery.min.js"></script>
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/bootcards/1.0.0/css/bootcards-desktop.min.css">
+
```
Comment (djmgit) [inline compareTwitterProfiles/index.html]: empty line

File: compareTwitterProfiles/index.html [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+    <script src="../../js/angular.min.js"></script>
+    <script src="../../js/angular-route.min.js"></script>
+    <script src="../../js/loklak.js"></script>
+    <link href="css/style.css" rel="stylesheet"/>
+    <link href="css/bootstrap.min.css" rel="stylesheet"/>
+    <script src="/js/angular.min.js"></script>
+    <script src="./script.js"></script>
+
```
Comment (djmgit) [inline compareTwitterProfiles/index.html]: here too

File: compareTwitterProfiles/index.html (hunk 8 above)
Comment (djmgit) [inline compareTwitterProfiles/index.html]: Also check indentation, html and head is in same indentation level

File: compareTwitterProfiles/index.html [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+                   </button>
+                 </span>
+               </div>
+            </div>
+            <br>
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-6">
+                        <div id="contactCard" style="pull-right">
+
```
Comment (djmgit) [inline compareTwitterProfiles/index.html]: empty line

File: compareTwitterProfiles/index.html [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+                                    </h4>
+                                </div>
+                                <div class="list-group-item">
+                                    <h4 class="list-group-item-heading">
+                                    <small>{{userData.screen_name}}</small>
+                                    </h4>
+                                </div>
+                            </div>
+                        </div>
+
```
Comment (djmgit) [inline compareTwitterProfiles/index.html]: empty line

File: compareTwitterProfiles/index.html [hunk 12]
```diff
@@ -0,0 +1,144 @@
...
+                    </div>
+                        <!-- <p>{{userData}}</p> -->
```
Comment (djmgit) [inline compareTwitterProfiles/index.html]: you may remove this

Comment (kavithaenair) [inline compareTwitterProfiles/index.html]: In the next iteration i would like to add more userdata here.

Comment (djmgit) [inline compareTwitterProfiles/index.html]: Okay :+1

**Comment by kavithaenair** [comment]
@djmgit Made the changes.

File: compareTwitterProfiles/index.html [hunk 13]
```diff
@@ -0,0 +1,135 @@
...
+                                ng-keyup="$event.keyCode == 13 && query1 != '' && query != '' ? Search() : null">
+                            </td>
+                        </tr>
+                    </table>
+                    <span class="input-group-btn">
+                    <button href='#' class="btn btn-default"
+                        type="button" id="button"
+                        ng-click="query != '' && query != '' ? Search() : null">Compare
+                    </button>
+                    </span>
```
Comment (djmgit) [inline compareTwitterProfiles/index.html]: Indentation problem, span and button on same level

Comment (kavithaenair) [inline compareTwitterProfiles/index.html]: Done @djmgit

File: compareTwitterProfiles/index.html [hunk 14]
```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html ng-app='TweetSearch'>
+    <head>
+        <link rel="icon" type="image/png" href="../../artwork/favicon.png">
+        <title>Comapre User Profile Page</title>
```
Comment (vibhcool) [inline compareTwitterProfiles/index.html]: typo error

**Comment by djmgit** [comment]
@kavithaenair please rebase

File: compareTwitterProfiles/index.html [hunk 15]
```diff
@@ -0,0 +1,135 @@
...
+        <nav class="navbar navbar-inverse navbar-fixed-top">
+            <div class="container-fluid">
+                <div class="navbar-header">
+                    <button type="button" class="navbar-toggle collapsed"
+                        data-toggle="collapse" data-target="#navbar"
+                        aria-expanded="false" aria-controls="navbar">
+                    <span class="sr-only">Toggle navigation</span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
```
Comment (djmgit) [inline compareTwitterProfiles/index.html]: indentation problem with spans

File: compareTwitterProfiles/index.html [hunk 16]
```diff
@@ -0,0 +1,135 @@
...
+                    </button>
+                    <a class="navbar-brand" href="http://loklak.org">
+                    <img src="/images/loklak_org.png" height="24" style="float:left;">
```
Comment (djmgit) [inline compareTwitterProfiles/index.html]: img is not indented

File: compareTwitterProfiles/index.html [hunk 17]
```diff
@@ -0,0 +1,135 @@
...
+                            type="button" id="button"
+                            ng-click="query != '' && query != '' ? Search() : null">Compare
+                        </button>
+                    </span>
+                </div>
+            </div>
+            <br>
+            <div class="container">
+            <div class="row">
+            <div class="col-md-6">
```
Comment (djmgit) [inline compareTwitterProfiles/index.html]: above two divs are not indented

File: compareTwitterProfiles/index.html [hunk 18]
```diff
@@ -0,0 +1,135 @@
...
+                <div id="navbar" class="navbar-collapse collapse">
+                    <ul class="nav navbar-nav navbar-right">
+                    </ul>
+                </div>
+            </div>
+        </nav>
+        <center>
+            <h1>Compare Twitter profiles</h1>
+            <div ng-app="TweetSearch" ng-controller="Controller">
+            <div class="form-group">
```
Comment (djmgit) [inline compareTwitterProfiles/index.html]: div is not indented

**Pull request merged**